release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -463,20 +463,25 @@ Leap second information is stored in an automatically generated `ConfigMap` reso
 
 For more information, see xref:../networking/ptp/configuring-ptp.adoc#ptp-configuring-dynamic-leap-seconds-handling-for-tgm_configuring-ptp[Configuring dynamic leap seconds handling for PTP grandmaster clocks].
 
-[id="ocp-4-17-nw-sriov-day1-nics"]
+[id="ocp-4-17-nw-sriov-day1-nics_{context}"]
 ==== NIC partitioning for SR-IOV devices (Generally Available)
 
 With this update, the ability to enable NIC partitioning for Single Root I/O Virtualization (SR-IOV) devices at install time is Generally Available.
 
 For more information, see xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#nw-sriov-dual-nic-con_preparing-to-install-on-bare-metal[NIC partitioning for SR-IOV devices].
 
-[id="ocp-4-17-nw-sriov-vfs-day2"]
+[id="ocp-4-17-nw-sriov-vfs-day2_{context}"]
 ==== Host network settings for SR-IOV VFs (Generally Available)
 
 With this update, the ability to update host network settings for Single Root I/O Virtualization (SR-IOV) network virtual functions in an existing cluster is Generally Available.
 
 For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc#virt-example-vf-host-services_k8s_nmstate-updating-node-network-config[Node network configuration policy for virtual functions].
 
+[id="ocp-4-17-user-defined-network_{context}"]
+==== User defined network segmentation (Technology Preview)
+
+With {product-title} 4.17, users can create multiple networks and declare them as primary or secondary networks for their workloads through the technology preview of the `UserDefinedNetwork` (UDN) custom resource definition (CRD). With UDN, users can isolate namespaces without configuring and managing complex network policies. For more information, see xref:./networking/multiple_networks/will add later when docs are live.
```

Comment:
```suggestion
For more information, see xref:./networking/multiple_networks/will add later when docs are live.
```
this looks weird? but guessing its a doc detail

Reply:
I agree with @tssurya unless this points to what ever documentation we are adding for UDN this looks like reminder to writer :)

Reply:
```
through the technology preview of the `UserDefinedNetwork` (UDN) custom resource definition (CRD) which is preferred or via network attachment definition.
```
@tssurya if we continue supporting NAD way we shall explicitely mention that as well imo. wdyt